src/components/WearablePreview/WearablePreview.tsx

```diff
@@ -45,6 +45,8 @@ export type WearablePreviewProps = {
 
 export type WearablePreviewState = {
   url?: string
+  isReady: boolean
```

Comment:
Is this used only here? Adding not optional properties could become a breaking change

Reply:
it's internal state tho, not a prop

Reply:
Maybe we could remove the export

Reply:
done ✅